DEVELOPING.md

```diff
@@ -25,6 +27,119 @@ are shared between multiple packages then they should be added to the Yarn const
 This is also where we define common peerDependencies and enforce a common version. These are packages such as React Native
 that our packages require but that we leave to the customer to have defined as explicit dependencies.
 
+## Adding new Native Modules
+
+If new packages needs to connect with native code there are few things to have in mind.
+The folder structure should be something like the following:
+
+```
+packages/
+└── my-new-package/
+    ├── ios/                 # containing an Xcode workspace/project
```

Comment:
were you able to get ios + android building using this structure without requiring a separate `native-src/` folder to wrap the package inside a test react native app?

Reply:
already discussed about it. ios component is able to compile using the test structure only